src/plugins/paced-transcript/index.js

```diff
@@ -0,0 +1,75 @@
+import { getCloudinaryUrl, extendCloudinaryConfig } from 'plugins/cloudinary/common';
+
+function pacedTranscript(config) {
+  const player = this;
+  const source = player.cloudinary.source();
+
+  const options = {
+    kind: config.kind || 'captions',
+    label: config.label || 'Captions',
+    default: config.default || true,
+    srclang: config.srclang || 'en',
+    src: config.src || getCloudinaryUrl(
+      source.publicId(),
+      extendCloudinaryConfig(player.cloudinary.cloudinaryConfig(), { resource_type: 'raw' }),
```

Comment:
i would extract the default value here to a constant, i think this would be easier to read.